src/.nuget/dir.props

```diff
@@ -149,6 +162,14 @@
       <Platform>arm64</Platform>
     </OfficialBuildRID>
   </ItemGroup>
+  <ItemGroup Condition="$(SupportedPackageOSGroups.Contains(';Android;'))">
+    <OfficialBuildRID Include="android.21-arm64">
+      <Platform>arm64</Platform>
+    </OfficialBuildRID>
+    <OfficialBuildRID Include="android.21-arm">
```

Comment:
Have you tested the build for Arm32 for Android? If so, your change for __DsitroRid should account for it as well.